data/cameras.xml

```diff
@@ -904,14 +907,16 @@
 	<Camera make="Canon" model="Canon EOS 7D" mode="sRaw1">
 		<ID make="Canon" model="EOS 7D">Canon EOS 7D</ID>
 		<Crop x="0" y="0" width="0" height="0"/>
-		<Sensor black="0" white="44200" iso_list="100 125 160 320 640 1250"/>
-		<Sensor black="0" white="54000"/>
+		<Sensor black="0" white="11400"/>
+		<Sensor black="0" white="9850" iso_list="100 125"/>
```

Comment:
That being said, (1<<16)/(1<<14) is 4 and 44200/11400 is ~3.8, so i think we just need to multiply the values by 4